src/metadata.js

```diff
@@ -0,0 +1,728 @@
+/**
+ * Created by valeriy.abornyev on 11/9/2015.
+ */
+    define(function() {
```

Comment:
Move to for example "test/data" folder 
Also use plain json, not AMD js file.
As a side effect you may need to provide url property for GraphExplorer